maven-release-manager/src/main/java/org/apache/maven/shared/release/phase/RunPerformGoalsPhase.java

```diff
@@ -99,7 +101,7 @@ private ReleaseResult runLogic( ReleaseDescriptor releaseDescriptor, ReleaseEnvi
         {
             ReleaseResult result = new ReleaseResult();
 
-            logInfo( result, "Simulating perform goals '" + getGoals( releaseDescriptor )
+            logInfo( result, "Simulating perform goals '" + buffer().strong( getGoals( releaseDescriptor ) )
```

Comment:
Why not `#mojo()`?

Reply:
we're using this style to list every phase that is being run
what goals will be invoked is a very different intent

Reply:
Right, agreed!